Dockerfile

```diff
@@ -21,13 +21,24 @@ ADD tests /docker/tests
 ADD music /docker/music
 ADD blender /docker/blender
 ADD nginx /docker/nginx
+ADD config /docker/config
 
 ## Install relevant packages
 
 RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get -y upgrade && apt-get install -y sudo lsb-release
 
 EXPOSE 80 8080 4002 6000 6001 6002 7000 7001 7002
 
+## Fix locate issues
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
```

Comment:
instead of adding another apt-get install (which does another update) just add 'locales' to the install instr. above

Then, as long as you're editing that line, put each package installed on it's own line (adding trailing `\` as necessary of course) - this makes adding/removing packages easier to read/review.